src/bin/rdcore/rootmap.rs

```diff
@@ -35,9 +35,13 @@ pub fn rootmap(config: RootmapConfig) -> Result<()> {
     let mut backing_devices = get_blkdev_deps_recursing(&device)?;
     backing_devices.push(device);
 
-    // for each of those, convert them to kargs
+    // for each of those (except crypt-integrity), convert them to kargs
     let mut kargs = Vec::new();
     for backing_device in backing_devices {
+        if Disk::new(&backing_device)?.is_luks_integrity()? {
+            continue;
+        }
```

Comment:
I think this would fit better as part of `device_to_kargs` where we would then return an `Ok(None)` like we do for some other no-op cases already. E.g. in the `else if blktype == "crypt" {` block, we can check if the device is `CRYPT-INTEGRITY`.